File: output/CodeLlama/avatar/Java/Python/codeforces_11_A.py
import sys

n, d = map(int, sys.stdin.readline().split())
nums = list(map(int, sys.stdin.readline().split()))
result = 0
a = 0
for i in range(1, n):
    a = (nums[i - 1] - nums[i])
    if a >= 0:
        result += a // d + 1
        nums[i] += (a // d + 1) * d
print(result)

File: output/ChatGPT/avatar/Java/C++/codeforces_11_A.cpp
#include <iostream>
using namespace std;

int main() {
    int n, d;
    cin >> n >> d;
    int nums[n];
    for (int i = 0; i < n; i++) {
        cin >> nums[i];
    }
    int result = 0;
    int a = 0;
    for (int i = 1; i < n; i++) {
        a = (nums[i - 1] - nums[i]);
        if (a >= 0) {
            result += a / d + 1;
            nums[i] += (a / d + 1) * d;
        }
    }
    cout << result << endl;
}